### ISSUE TITLE
Is this script compatible to compile libssh 0.9.6 for ios / macOS?

### ISSUE THREAD

**Issue by insinfo** [Frugghi/iSSH2#16]
Is this script compatible to compile libssh 0.9.6 for ios / macOS?
Hey I made a libssh binding on dart to be able to use libssh on windows and linux with flutter, now I want to use it on ios and macOS too, but I don't know how to compile libssh for ios

https://pub.dev/packages/libssh_binding
https://www.libssh.org/

**Comment by Jan-E** [comment]
No, it is not. The script builds SSH2 for iOS and macOS. Just yesterday I upgraded my libraries for iOS and Mac Catalyst, using SSH2 version 1.10.0 and OpenSSL 1.1.1m. See https://www.libssh2.org/libssh2-vs-libssh.html for a comparison between SSH and SSH2.

**Comment by insinfo** [comment]
libssh2 does not implement recursive copying via SCP, therefore libssh2 is extremely slow for bulk file copying.

https://github.com/libssh2/libssh2/issues/646
